src/core_plugins/kibana/public/management/sections/settings/advanced_import_controls.js

```diff
@@ -0,0 +1,93 @@
+import { uiModules } from 'ui/modules';
+import { saveAs } from '@spalger/filesaver';
+import { AdvancedImportState } from 'plugins/kibana/management/sections/settings/advanced_import_state.js';
+import advancedImportControlsTemplate from 'plugins/kibana/management/sections/settings/advanced_import_controls.html';
+
+uiModules.get('apps/management')
+  .directive('advancedImportControls', function (config, Notifier, Private) {
+    const notify = new Notifier({ location: 'Advanced Settings' });
+    const advancedImportState = Private(AdvancedImportState);
+
+    return {
+      restrict: 'E',
+      template: advancedImportControlsTemplate,
+      controllerAs: 'advancedImportControls',
+      bindToController: true,
+      controller: class AdvancedImportController {
+        constructor() {
+          this.importState = advancedImportState;
```

Comment:
I'd prefer to see you use `this.importState` after this point. That way the alias is consistent here and in the template.